<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Playback</class>
 <widget class="QWidget" name="Main_Playback">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>251</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;font-size:15px</string>
    </property>
    <property name="resizeMode">
     <enum>QListView::Adjust</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>781</width>
      <height>491</height>
     </rect>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_butt">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_butt">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>500</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>下一页&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="month_butt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>按月显示</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="day_butt">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>按天显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="page_label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
